eng/scripts/Language-Settings.ps1

```diff
@@ -216,18 +216,30 @@ function Get-DocsMsPackageName($packageName, $packageVersion) {
 #   registry = "<url>";
 #   ...
 # }
-function ValidatePackagesForDocs($packages, $DocValidationImageId) {
+function GetResult($success, $package, $output) { 
+  return @{ Success = $success; Package = $package; Output = $output }
+}
+
+function RexToolValidation($package, $output) {
+  $validateOutput = type2docfx $package $output
+  if ($LASTEXITCODE -ne 0) {
+    LogError "Package $package failed the rex tool validation. Check error details by runing the following command:"
```

Comment:
Calling LogError here might cause the script to fail fast depending on what the error mode it set to. Make sure you test a failure here in the pipeline to make sure this correctly works.